pandas/tests/arithmetic/conftest.py

```diff
@@ -2,6 +2,7 @@
 import pytest
 
 import pandas as pd
+from pandas import Float64Index, Int64Index, RangeIndex, UInt64Index
```

Comment:
This is to be consistent with one of the docstrings, which uses `Float64Index` rather than `pd.Float64Index`